packages/simple-hub/package.json

```
@@ -39,7 +39,7 @@
     "lint-staged": "10.0.4",
     "nodemon": "1.19.4",
     "npm-run-all": "4.1.5",
-    "pino-pretty": "3.5.0",
+    "pino-pretty": "^4.0.0",
```

I believe we aim to have exact dependencies
```suggestion
    "pino-pretty": "4.0.0",
```